src/user/auth/signup-form/signup-form.js

```diff
@@ -319,6 +352,7 @@ class SignUpForm extends Component {
 // map state to props
 const mapStateToProps = (state) => ({
```

Comment:
@tushar912 With this implementation, if we try to register with an already registered email address, it will still say that the registration was successful. This shouldn't be the case. It would be awesome if you could display some sort of error message in such an instance.  You can access the error through the redux state passed down to this component through props.